allure-report-face/src/components/pane-set/styles.css

```diff
@@ -64,3 +63,7 @@
     }
   }
 }
+
+.pane + .pane {
+  box-shadow: 0 0 3px #eee;
+}
```

Comment:
This shadows here are useless. 

Also, since you added a shadow here, you should add it to more similar places to make this update consistent.

![image](https://cloud.githubusercontent.com/assets/812240/15544632/ad82c700-2299-11e6-91ee-08e0ac048158.png)